.github/workflows/integration_tests.yaml

```diff
@@ -71,6 +75,22 @@ jobs:
       - name: Check paas
         id: test_check_paas
         run: kubemarine check_paas -c ${{ env.cluster_yaml_file }} --dump-location ./results/check_paas_dump/
+      - name: Collect Kubernetes logs
+        run: |
+          sudo journalctl -u kubelet -u kube-proxy -u kube-apiserver -u kube-controller-manager -u kube-scheduler > ./results/kube-logs.txt
+          sudo journalctl -u kubelet -u kube-proxy -u kube-apiserver -u kube-controller-manager -u kube-scheduler -o json > ./results/kube-logs.json
+          sudo journalctl -u etcd > ./results/etcd-logs.txt
+          sudo journalctl -u etcd -o json > ./results/etcd-logs.json
+          sudo journalctl -u coredns > ./results/coredns-logs.txt
+          sudo journalctl -u coredns -o json > ./results/coredns-logs.json
+          sudo journalctl > ./results/all-logs.txt
```

Comment:
I think in case we collect ALL logs , no need to collect kubelet/etcd/coredns specific additionally

Reply:
Yes, Collecting all system logs using sudo journalctl will provide you with a comprehensive set of logs, However, in certain scenarios, it can be beneficial to collect logs specifically for Kubernetes-related components like kubelet, etcd, and coredns. These logs can provide more focused information related to Kubernetes-specific issues.